pandas/tests/io/excel/test_writers.py

```diff
@@ -61,7 +60,6 @@ def set_engine(engine, ext):
     [
         pytest.param(".xlsx", marks=[td.skip_if_no("openpyxl"), td.skip_if_no("xlrd")]),
         pytest.param(".xlsm", marks=[td.skip_if_no("openpyxl"), td.skip_if_no("xlrd")]),
-        pytest.param(".xls", marks=[td.skip_if_no("xlwt"), td.skip_if_no("xlrd")]),
```

Comment:
Guessing xlrd will be removed too but not part of this PR?

Reply:
I _think_ based on https://github.com/pandas-dev/pandas/pull/38317 that xlrd will continued to be supported since it's just for reading (although the package isn't maintained).

Reply:
I'm good with keeping xlrd. However, I'm thinking we should warn users on any use that it is not maintained and bump the minimum version to 2.0 (so it can only be used to read xls files).

Reply:
That sounds good to me. Probably good to warn and bump of the min version in a separate PR.